tools/venv_transfer_commands.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+# Please don't try and run this script directly!
+ssh -o StrictHostKeyChecking=no cvmfs.pycbc@cvmfs-software.ligo.caltech.edu "sudo -u repo.software cvmfs_server transaction"
+ssh -o StrictHostKeyChecking=no cvmfs.pycbc@cvmfs-software.ligo.caltech.edu "mkdir -p /cvmfs/software.igwn.org/pycbc/${ENV_OS}/virtualenv/pycbc-${SOURCE_TAG}"
```

Comment:
It is probably better to use a single `ssh` call for both commands here, and also add the `-t 300` option to the `cvmfs_server transaction` command to allow 5 minutes for an in-progress transaction (from me, probably) to finish:

```suggestion
ssh -o StrictHostKeyChecking=no cvmfs.pycbc@cvmfs-software.ligo.caltech.edu <<EOF
sudo -u repo.software cvmfs_server transaction -t 300 software.igwn.org
mkdir -p /cvmfs/software.igwn.org/pycbc/${ENV_OS}/virtualenv/pycbc-${SOURCE_TAG}
EOF
```